raiden/utils/packing.py

```diff
@@ -61,22 +65,16 @@ def pack_balance_proof_update(
 
 
 def pack_reward_proof(
-    canonical_identifier: CanonicalIdentifier,
+    chain_id: ChainID,
     reward_amount: TokenAmount,
-    nonce: Nonce,
     monitoring_service_contract_address: Address,
```

Comment:
nit (feel free to ignore): Would be nice to order the arguments in the way they are used in the signature.